### PR TITLE
gpu: add mentions about data center gpu support

### PR DESCRIPTION
**Pull request by tkatila**
Signed-off-by: Tuomas Katila <tuomas.katila@intel.com>

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]
/lgtm

File: cmd/gpu_plugin/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Table of Contents
 ## Introduction
 
 Intel GPU plugin facilitates Kubernetes workload offloading by providing access to
-Intel discrete (Xe) and integrated GPU HW device files.
+Intel discrete (including Intel® Data Center GPU Flex Series) and integrated GPU HW device files.
```
Comment (eero-t) [inline cmd/gpu_plugin/README.md:19]: Maybe "discrete (...branding...) and integrated Intel GPU device files"?

(move "Intel" to front of "GPU" like in main README.)

**Comment by eero-t** [comment]
When given plugin feature, or NFD label, is specific to given HW feature (SR-IOV, tiles...), it would be good to mention required HW series also in the README.

File: cmd/gpu_plugin/README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Table of Contents
 ## Introduction
 
 Intel GPU plugin facilitates Kubernetes workload offloading by providing access to
-Intel discrete (Xe) and integrated GPU HW device files.
+discrete (including Intel® Data Center GPU Flex Series) and integrated Intel GPU device files.
```
Comment (eero-t) [inline cmd/gpu_plugin/README.md:19]: Maybe add also "... supported by the host kernel."?

Comment (tkatila) [reply cmd/gpu_plugin/README.md:19]: In that case, I'd say: .. integrated Intel GPU devices supported by the host kernel." ? Without the "files".

Comment (eero-t) [reply cmd/gpu_plugin/README.md:19]: Yes, that's better!

Note: this PR is fine as-is, I can add that kernel note later, in my prerequisites PR.

**Comment by tkatila** [comment]
Feel free to review again.

**Review by uniemimu — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**